File: tech5.rb
i = 0
print "Enter a number: "
j = gets.to_f
k = (j + 13) / 27.0

while k < 10
  k += 1
  i = 3 * k - 1
  puts "i: #{i}"
  puts "k: #{k}"
  puts "j: #{j}"
  puts
end

File: tech5.py
i = 0
j = float(input("Enter a number: "))
k = (j + 13) / 27

while (k < 10):
    k += 1;
    i = 3 * k - 1;
    print("i: " + str(i));
    print("k: " + str(k));
    print("j: " + str(j));
    print();